test/test_basebackup.py

```diff
@@ -625,12 +625,13 @@ def test_handle_site(self, pghoard):
 
         # adjust basebackup interval to be slightly longer than what this
         # basebackup took and make sure it's not retriggered
```

Comment:
It would make reviewing this change much easier if this comment would explain why we need to do all of this :-)

Reply:
I tried to adjust some comments, let me know what you think.